pom.xml

```diff
@@ -472,22 +472,42 @@
         </dependencies>
         <!-- Specifying configuration here will take effect on the execution of "mvn checkstyle:checkstyle",
              but will not take effect on the execution of "mvn site"  -->
-        <configuration>
-          <configLocation>${project.basedir}/config/checkstyle_sevntu_checks.xml</configLocation>
-          <failOnViolation>true</failOnViolation>
-          <includeResources>false</includeResources>
-          <includeTestResources>false</includeTestResources>
-          <logViolationsToConsole>true</logViolationsToConsole>
-          <maxAllowedViolations>0</maxAllowedViolations>
-          <violationSeverity>error</violationSeverity>
-          <propertyExpansion>project.basedir=${project.basedir}</propertyExpansion>
-          <sourceDirectory>${project.basedir}/src</sourceDirectory>
-          <excludes>**/it/resources/**/*,**/test/resources/**/*,**/test/resources-noncompilable/**/*</excludes>
-        </configuration>
         <executions>
           <execution>
             <id>sevntu-checkstyle-check</id>
             <phase>verify</phase>
+            <configuration>
+              <configLocation>${project.basedir}/config/checkstyle_sevntu_checks.xml</configLocation>
+              <failOnViolation>true</failOnViolation>
+              <includeResources>false</includeResources>
+              <includeTestResources>false</includeTestResources>
+              <logViolationsToConsole>true</logViolationsToConsole>
+              <maxAllowedViolations>0</maxAllowedViolations>
+              <violationSeverity>error</violationSeverity>
+              <propertyExpansion>project.basedir=${project.basedir}</propertyExpansion>
+              <sourceDirectory>${project.basedir}/src</sourceDirectory>
+              <excludes>**/it/resources/**/*,**/test/resources/**/*,**/test/resources-noncompilable/**/*</excludes>
```

Comment:
please do not reply to us as "Will" , You will do it for sure. just reply to us when it is  "done".